collect_app/src/main/AndroidManifest.xml

```diff
@@ -33,7 +33,7 @@ the specific language governing permissions and limitations under the License.
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
unused change

Reply:
Could you explain what you mean?

Reply:
@huangyz0918 This change although simple helps to maintain the style of the permissions so I think it's fine.